public/app/plugins/datasource/grafana-azure-monitor-datasource/resourcePicker/resourcePickerData.test.ts

```diff
@@ -30,21 +30,22 @@ describe('AzureMonitor resourcePickerData', () => {
     it('returns only subscriptions at the top level', async () => {
       const results = await resourcePickerData.getResourcePickerData();
 
-      expect(results.map((v) => v.id)).toEqual(['/subscriptions/abc-123', '/subscription/def-456']);
+      expect(results.map((v) => v.id)).toEqual(['/subscriptions/abc-123', '/subscriptions/def-456']);
```

Comment:
Maybe you can add in the test above ( line 27) a check that verifies that the query contains `| project resourceGroupURI=id, resourceGroupName=name, resourceGroup, subscriptionId`? Just to have a test that covers the new code